controlplane/pkg/tests/nsmd_plugin_test.go

```diff
@@ -0,0 +1,61 @@
+package tests
+
+import (
+	"context"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	pluginsapi "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/plugins"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/plugins"
+)
+
+func TestPluginReRegistration(t *testing.T) {
+	g := NewWithT(t)
+	var err error
+
+	const pluginName = "dummy-plugin"
+
+	registryDir := path.Join(os.TempDir(), "networkservicemesh")
+	if err = os.MkdirAll(registryDir, os.ModeDir|os.ModePerm); err != nil {
+		logrus.Errorf("Failed to create socket folder for Plugin Registry: %v", err)
+		return
+	}
+	defer func() { _ = os.RemoveAll(registryDir) }()
+
+	registryPath := path.Join(registryDir, path.Base(pluginsapi.PluginRegistrySocket))
+
+	pluginRegistry := plugins.NewPluginRegistry(registryPath)
+	if err = pluginRegistry.Start(context.Background()); err != nil {
+		logrus.Errorf("Failed to start Plugin Registry: %v", err)
+		return
+	}
+
+	defer func() {
+		if err = pluginRegistry.Stop(); err != nil {
+			logrus.Errorf("Failed to stop Plugin Registry: %v", err)
+		}
+	}()
+
+	plugin := &dummyConnectionPlugin{
+		prefixes: []string{"10.10.1.0/24"},
+	}
+	services := make(map[pluginsapi.PluginCapability]interface{}, 1)
+	services[pluginsapi.PluginCapability_CONNECTION] = plugin
+
+	if err = plugins.StartPlugin(pluginName, registryPath, services); err != nil {
+		logrus.Fatalln("Failed to start first instance of dummy plugin", err)
```

Comment:
`logrus.Fatalln` will fail the test, but we won't get expected output like:
```
--- FAIL: TestPluginReRegistration (0.00s)
    nsmd_plugin_test.go:51: ...reason...
FAIL
```
so please use `t.Fatalf("...")` or similar methods

Reply:
- [ ] logrus.Fatalln => logrus.Fatalf